src/server/journal/types.h

```diff
@@ -44,6 +48,35 @@ struct Entry {
   uint64_t expire_ms = 0;  // 0 means no expiry.
 };
 
+// This struct represents a single journal entry.
+// Those are either control instructions or commands.
+struct EntryNew {  // Called this "New" because I can't delete the old neither replace it partially
```

Comment:
I called this New because we need both, because this PR contains only the declaration & read+write utils